app/src/androidTest/java/com/appunite/loudius/LoginScreenTest.kt

```diff
@@ -46,12 +55,24 @@ class LoginScreenTest {
 
     @Test
     fun whenTheLoginScreenIsVisibleThenTheLogInButtonIsVisible() {
+        Intents.init()
+        intending(hasAction(Intent.ACTION_VIEW))
+            .respondWith(Instrumentation.ActivityResult(Activity.RESULT_OK, null))
+
         composeTestRule.setContent {
             LoudiusTheme {
                 LoginScreen()
             }
         }
 
-        composeTestRule.onNodeWithText("Log in").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Log in").performClick()
+
+        intended(
+            allOf(
+                hasAction(Intent.ACTION_VIEW),
+                hasData("https://github.com/login/oauth/authorize?client_id=91131449e417c7e29912&scope=repo"),
```

Comment:
Question: Do we want pass `client_id` like that? Or maybe we can pass whole url as value? I think something like `private const val AUTH_URL = "https://github.com/login/oauth/authorize?client_id=91131449e417c7e29912&scope=repo"` would be more readable.